### DISCUSSION TITLE
Failed to start gauge API: Timed out connecting to java

### DISCUSSION THREAD

**Discussion by patricia2193** [getgauge/gauge#2098] [General]
Hello, 

I have a problem that started a couple of days ago. I cannot start the tests as they get stuck at "Testing started" and then I get "Failed to start gauge API: Timed out connecting to java". 

Could you please help me out?

This is the config: 
Key                             Value
------------------------------------------------------------------
allow_insecure_download         true
check_updates                   true
gauge_repository_url            https://downloads.gauge.org/plugin
ide_request_timeout             30000
plugin_connection_timeout       10000
plugin_kill_timeout             8000
runner_connection_timeout       200000
runner_request_timeout          500000

The plugins: 
Gauge version: 1.2.1

Plugins
-------
html-report (4.0.12)
java (0.7.15)
screenshot (0.1.0)
xml-report (0.2.3)
xray-report (0.3.2)

**Comment by sriv** [comment]
hi - is this something that happens for a specific project or do you see this behaviour across all gauge-java projects in your machine?
